sumpy/p2p.py

```diff
@@ -472,7 +520,7 @@ def get_kernel(self):
             fixed_parameters=dict(
                 dim=self.dim,
                 nstrengths=self.strength_count,
-                nkernels=len(self.kernels)),
+                nkernels=len(self.target_kernels)),
```

Comment:
The variable name `nkernels` should be changed. `ntarget_kernels`? (Not that I like this very much.) `noutputs`? (In preparation of introducing "outputs" that aren't kernels?)

(Here and elsewhere)

Reply:
Used noutputs